Defs/ThingDefs_Misc/Weapons_Turrets.xml

```diff
@@ -328,7 +328,7 @@
         <defaultProjectile>Bullet_90mmCannonShell_HEAT</defaultProjectile>
         <warmupTime>4.55</warmupTime>
         <minRange>16</minRange>
-        <range>86</range>
+        <range>100</range>
```

Comment:
Too much. Bear in mind, this is range for engaging a man-sized target, not an armored vehicle. If we really want to increase it (and I don't really think that it's in need of an increase), most I'd want to see it move is 88 to 92 range.